enterprise/frontend/src/metabase-enterprise/whitelabel/lib/whitelabel.js

```diff
@@ -20,20 +20,11 @@ const JS_COLOR_UPDATORS_BY_COLOR_NAME = {};
 const RANDOM_COLOR = Color({ r: 0xab, g: 0xcd, b: 0xed });
 
 function colorScheme() {
-  // FIXME: Ugh? initially load public setting as "application_color" but if the admin updates it
```

Comment:
This is not true anymore. The setting is called `application-colors` on the backend, there is no `application_colors`